deploy/qpc

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 mkdir -p ~/.config/qpc ~/.local/share/qpc
-podman run -it  -v ~/.config/qpc:/root/.config/qpc \
-                -v ~/.local/share/qpc:/root/.local/share/qpc \
-                qpc "$@"
+podman run -it --rm \
+    -v ~/.config/qpc:/root/.config/qpc:z \
+    -v ~/.local/share/qpc:/root/.local/share/qpc:z \
```

Comment:
The Macs are not happy with the \:z at the end. Maybe differentiate with a $(uname) == "Darwin" case.

Reply:
Sure thing
just out of curiosity, could you share the error message?

Reply:
```
Error: preparing container d673da3623e2491ccc6938287133a151dc79114d4253c7ff6931b875ca1d9e84 for attach: lsetxattr /Users/abellotti/.config/qpc: operation not supported
```